app/page.tsx

```diff
@@ -1,12 +1,13 @@
 import { Card } from '@/components/Card';
-import { Grid, Timeline } from '@/components/Grid';
+import { Grid } from '@/components/Grid';
 import { Heading, Paragraph, Text } from '@/components/Typography';
+import { TimelineSection } from '@/components/PageSections';
```

Comment:
I'm starting to break these out into sections so this `page.tsx` file doesn't get too long